include/class.page.php

```diff
@@ -153,10 +153,11 @@ function delete() {
 
         if (!parent::delete())
             return false;
-
-        return Topic::objects()
-            ->filter(array('page_id'=>$this->getId()))
-            ->update(array('page_id'=>0));
```

Comment:
This didn't appear to be relevant because we do not allow the Agent to delete a Page if a Topic is using it.

Reply:
Why the need for else block below?

Reply:
I went ahead and took the else out.